moveit_ros/perception/occupancy_map_monitor/include/moveit/occupancy_map_monitor/occupancy_map_monitor.h

```diff
@@ -152,12 +155,9 @@ class OccupancyMapMonitor
 
   std::size_t mesh_handle_count_;
 
-  ros::NodeHandle root_nh_;
-  ros::NodeHandle nh_;
-  ros::ServiceServer save_map_srv_;
-  ros::ServiceServer load_map_srv_;
-
   bool active_;
+
+  // rclcpp::Publisher<octomap_msgs::msg::Octomap> octree_binary_pub_;
```

Comment:
Why is this commented out?

Reply:
I was not using it, it was a still WIP, fixed it on https://github.com/AcutronicRobotics/moveit2/commit/92c28f9a441ae4a849d83af31223d35cfc4efdd8